website/integrations/services/xen-orchestra/index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Xen Orchestra
+---
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## What is Xen Orchestra
+
+> Xen Orchestra provides a user friendly web interface for every Xen based hypervisor (XenServer, xcp-ng, etc.).
+>
+> -- https://xen-orchestra.com/
+
+:::note
+Xen Orchestra offers authentication plugins for OpenID Connect, SAML and LDAP. This guide is usind the OpenID Connect plugin.
+If you are using the Xen Orchestra Appliance, the OIDC Plugin should be present. If you are using Xen Orchestra compiled from sources, make sure the plugin `auth-oidc` is installed.
+:::
+
+## Preparation
+
+The following placeholders will be used:
+
+-   `xenorchestra.company` is the FQDN of the Xen Orchestra instance.
+-   `authentik.company` is the FQDN of the authentik install.
+
+## authentik configuration
+
+### 1. Provider
+
+Under _Providers_, create an OAuth2/OpenID provider with these settings:
+
+-   Name: Provider for XenOrchestra
+-   Authorization Flow: Select one of the availible Flows.
+-   Client type: Confidental
+-   Redirect URIs/Origins: `https://xenorchestra.company/signin/oidc/callback`
+
+Take note of the Client ID and the Client Secret, because we need them for the Configuration in Xen Orchestra.
+
+### 2. Application
+
+Create an application with the following details:
+
+-   Slug: `xenorchestra` (If you want to choose a different slug, your URLs for the Xen Orchestra Configuration may vary)
+-   Provider: Select the one we have created in Step 1
+-   Set the Launch URL to `https://xenorchestra.company/`.
+
+Optionally apply access restrictions to the application.
+
+## Xen Orchestra configuration
+
+Xen Orchestra allows the configuration of the OpenID Connect authentication in the plugin-section.
+All of the URLs metioned below can be copied & pasted from authentik (Applications -> Providers -> the provider created earlier)
+
+1. Navigate to Settings -> Plugins
+2. Scroll to "auth-oidc" and click on the "+ Icon" on the right hand side.
+3. Set the `Auto-discovery URL` to `https://authentik.company/application/o/xenorchestra/.well-known/openid-configuration`
```

Comment:
```suggestion
3. Set the `Auto-discovery URL` to `https://authentik.company/application/o/xenorchestra/.well-known/openid-configuration`.
```

Reply:
Up to you, but where possible, you can simplify the docs by grouping the stps all into a single step, if they are done on the same UI page in Xen Orchestra. (Sorry I don't have access to the XO Ui or I would make a more exact suggestion.) Basically, if a group of settings are on the same UI, you can format it like this:

3. On the <**xyz page**>, define the following configuration values:
      - Auto-discovery URL: `https://authentik.company/application/o/xenorchestra/.well-known/openid-configuration`
      - Client indentifier (key): the Client ID from your notes
      - Client secret: the Client Secret from your notes
      - etc etc

Reply:
Thats a great idea. I will change that 👍 

There's a screenshot of the configuration page in Xen Orchestra. All settings can be accessed within the same page.
![Screenshot XOA](https://github.com/goauthentik/authentik/assets/60177408/e911ba86-dd59-4af1-bc1d-97dde2a8395f)